docs/source/techniques/ReflectometryILL.rst

```diff
@@ -0,0 +1,519 @@
+.. _ReflectometryILL:
+
+=======================================================
+Data reduction for reflectometry instruments at the ILL
+=======================================================
+
+.. contents:: Table of contents
+    :local:
+
+There is one algorithm providing the entry point and steering the reduction workflow for both FIGARO and D17:
+
+:ref:`algm-ReflectometryILLAutoProcess`
+     Controls the reduction workflow of both direct and reflected beams from loading, through corrections, to final reflectivity curve as a function of momentum exchange.
+
+This algorithm depends on the number of dedicated ILL algorithms handling loading, corrections, and converting data:
+
+:ref:`algm-LoadILLReflectometry`
+     Loads D17 and FIGARO NeXus data, fits the foreground position and places the instrument such that the foreground falls at :math:`2\theta = 0`
+
+:ref:`algm-ReflectometryILLPreprocess`
+     Loads, merges, normalizes, subtracts background, and calculates gravity correction for FIGARO.
+
+:ref:`algm-ReflectometryILLSumForeground`
+     Sums foreground pixels in selected summation mode, optionally converting to reflectivity.
+
+:ref:`algm-ReflectometryILLPolarizationCor`
+     Performs polarization efficiency corrections.
+
+:ref:`algm-ReflectometryILLConvertToQ`
+     Converts a reflectivity workspace from wavelength to momentum transfer.
+
+Together with other algorithms and services provided by the Mantid framework, the reduction algorithms can handle a number of reduction scenarios. If this proves insufficient, however, the algorithms can be accessed using Python. Before making modifications it is recommended to copy the source files and rename the algorithms as not to break the original behavior.
+
+This document tries to give an overview on how the algorithms work together via Python examples. Please refer to the algorithm documentation for details of each individual algorithm.
+
+A description of the usage of the algorithms for the D17 and FIGARO data reduction is presented along with several possible workflows, depending on summation type, the source of the angle (detector, sample, user-defined), and whether the neutrons are polarized.
+
+Reduction basics
+================
+
+A very basic data reduction would include a direct and a reflected beam measurement of the sample at one angle position, follow the following steps:
+
+#. Preprocessing of direct and reflected beams in :ref:`ReflectometryILLConvertToQ <algm-ReflectometryILLConvertToQ>`
```

Comment:
Is the algorithm name correct here? I wondered if it should be ReflectometryILLPreProcess looking at the workflow diagram

Reply:
Yes indeed, this is certainly an error.